editor/opening_hours_ui.hpp

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+#include "std/set.hpp"
+#include "std/vector.hpp"
+
+#include "3party/opening_hours/opening_hours.hpp"
+
+namespace editor
+{
+namespace ui
+{
+
+using TOpeningDays = set<osmoh::Weekday>;
```

Comment:
One particular compiler on a particular machine? Are you kiddin me? Please, verify it on all compilers (especially on compilers from the future) and on all machines in the universe.